src/schema/prov.yaml

```diff
@@ -76,6 +77,9 @@ slots:
       - range: OmicsProcessing
     mappings:
       - prov:wasGeneratedBy
+    structured_pattern:
```

Comment:
Same comments as on slot was_generated_by on Class DataObject. 'wf' should be removed, WorkflowExeuctionActivity is abstract. id_version should be allowed on workflow activity subclasses but not OmicsProcessing records so I think this should be re-written like this
syntax: "{id_nmdc_prefix}:(wfmag|wfmb|wfmgan|wfmgas|wfmsa|wfmp|wfmt|wfmtan|wfmtas|wfnom|wfrbt|wfrqc)-{id_shoulder}-{id_blade}{id_version}$ | {id_nmdc_prefix}:omprc-{id_shoulder}-{id_blade}$"
Can we just define this globally on the slot and then remove the local definition on Class DataObject?